<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sancomp v1.2b</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>731</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>&amp;Customize</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>321</width>
       <height>310</height>
      </rect>
     </property>
     <property name="title">
      <string>Laminate</string>
     </property>
     <widget class="QPushButton" name="calcular_f">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>270</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>133</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>75</y>
        <width>133</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Young's Modulus (MPa)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>133</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Poisson's ratio</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>181</x>
        <y>75</y>
        <width>133</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Shear Modulus (MPa)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>31</y>
        <width>133</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Density (Kg/m³)</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="ex">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>92</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>47</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="gxy">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>92</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="density_f">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>48</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="vxy">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>138</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>181</x>
        <y>121</y>
        <width>133</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Ult. Shear Strenght (MPa)</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="shear_st">
      <property name="geometry">
       <rect>
        <x>181</x>
        <y>139</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="tensile_st">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>185</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>167</y>
        <width>133</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Ult. Tensile Strenght (MPa)</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="compressive_st">
      <property name="geometry">
       <rect>
        <x>181</x>
        <y>186</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>181</x>
        <y>168</y>
        <width>133</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Ult. Compr. Strenght (MPa)</string>
      </property>
     </widget>
     <widget class="QLabel" name="lami_msg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_12">
     <property name="geometry">
      <rect>
       <x>391</x>
       <y>15</y>
       <width>321</width>
       <height>310</height>
      </rect>
     </property>
     <property name="title">
      <string>Core</string>
     </property>
     <widget class="QPushButton" name="calcular_c">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>270</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>133</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>75</y>
        <width>133</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Young's Modulus (MPa)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>31</y>
        <width>133</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Density (Kg/m³)</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="core_ex">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>92</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="core_name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>47</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="density_c">
      <property name="geometry">
       <rect>
        <x>179</x>
        <y>48</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="core_msg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="core_ex_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>91</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>179</x>
        <y>74</y>
        <width>133</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Ult. Shear Strenght (MPa)</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="core_ex_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>136</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>179</x>
        <y>119</y>
        <width>133</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Ult. Compr. Strenght (MPa)</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="core_ex_4">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>137</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>133</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Ult. Tensile Strenght (MPa)</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>576</x>
       <y>340</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Abrir Mech-GComp</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage1">
    <attribute name="title">
     <string>&amp;Geometry and Loads</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>331</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>Geometry</string>
     </property>
     <widget class="QComboBox" name="cb_laminate">
      <property name="geometry">
       <rect>
        <x>77</x>
        <y>20</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Choose laminate...</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>20</y>
        <width>86</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Laminate</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>76</y>
        <width>86</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Beam type</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_beam">
      <property name="geometry">
       <rect>
        <x>77</x>
        <y>76</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Choose beam...</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Simple-Simple / Point</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Simple-Simple / Uniform</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Free-Fixed / Point</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Free-Fixed / Uniform</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fixed-Fixed / Point</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fixed-Fixed / Uniform</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cb_core">
      <property name="geometry">
       <rect>
        <x>77</x>
        <y>48</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Choose core...</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>48</y>
        <width>86</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Core type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>104</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>F(N), p0(N/m)</string>
      </property>
     </widget>
     <widget class="QLabel" name="pic_sandwich">
      <property name="geometry">
       <rect>
        <x>203</x>
        <y>150</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>image/sec2.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>238</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>L (mm)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>210</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>b (mm)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>182</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>H (mm)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="calcular_g">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>153</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>h (mm)</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="pic_beam">
     <property name="geometry">
      <rect>
       <x>359</x>
       <y>50</y>
       <width>350</width>
       <height>250</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="load">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>122</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="h">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>170</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="H">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>200</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="b">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>228</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="l">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>258</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="message">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Beam Analysis</string>
    </attribute>
    <widget class="QLabel" name="resultado">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>331</width>
       <height>261</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
    <widget class="QPushButton" name="help">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>350</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Help</string>
     </property>
    </widget>
    <widget class="QLabel" name="resultado_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>331</width>
       <height>261</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>INPUTS</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>OUTPUTS</string>
     </property>
    </widget>
    <widget class="QLabel" name="fs_msg">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>18</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Panel Analysis</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>OUTPUTS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_2">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
    <widget class="QPushButton" name="help_2">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>350</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Help</string>
     </property>
    </widget>
    <widget class="QLabel" name="resultado_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>331</width>
       <height>261</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="resultado_4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>331</width>
       <height>261</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>INPUTS</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>name</tabstop>
  <tabstop>gxy</tabstop>
  <tabstop>ex</tabstop>
  <tabstop>density_f</tabstop>
  <tabstop>vxy</tabstop>
  <tabstop>shear_st</tabstop>
  <tabstop>tensile_st</tabstop>
  <tabstop>compressive_st</tabstop>
  <tabstop>calcular_f</tabstop>
  <tabstop>core_name</tabstop>
  <tabstop>density_c</tabstop>
  <tabstop>core_ex</tabstop>
  <tabstop>calcular_c</tabstop>
  <tabstop>cb_laminate</tabstop>
  <tabstop>cb_core</tabstop>
  <tabstop>cb_beam</tabstop>
  <tabstop>load</tabstop>
  <tabstop>h</tabstop>
  <tabstop>H</tabstop>
  <tabstop>b</tabstop>
  <tabstop>l</tabstop>
  <tabstop>calcular_g</tabstop>
  <tabstop>save</tabstop>
  <tabstop>help</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
